src/js/plugin/external/PluginHelper.js

```diff
@@ -1,27 +0,0 @@
-import PluginDispatcher from "./PluginDispatcher";
```

Comment:
Why do I end up exactly in the same "jQuery"-global-scope? I mean sure, those object live in the same scope and are kinda global as they are "singletons" but I'm able to explicitly import them from the SDK. 

```
var {PluginMountPoints} = window["marathonPluginInterface"];
var {PluginAppsActions} = marathonPluginInterface;
marathonPluginInterface.PluginEvents.APPS_STORE_CHANGE
```
```
import import {PluginHelper, PluginMountPoints} from "marathon-ui-plugin-sdk";
```

I prefer the latter.